willisapi_client/services/metadata/archive.py

```
@@ -0,0 +1,83 @@
+):
+    """Report the CSV archive outcome and parsed row counts to the server."""
+    if not record_id:
+        return
+    try:
+        wc = WillisapiClient(env=env)
+        payload = {
+            "status": status,
+            "successful_rows": successful_rows,
+            "failed_rows": failed_rows,
+        }
+        res = requests.patch(
```

Ruff (0.15.15)

[error] Probable use of `requests` call without timeout

(S113)